python/cugraph/opg/link_analysis/mg_pagerank_wrapper.pyx

```diff
@@ -0,0 +1,79 @@
+#
+# Copyright (c) 2020, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from cugraph.structure.utils_wrapper import *
+from cugraph.opg.link_analysis cimport mg_pagerank as c_pagerank
+import cudf
+from cugraph.structure.graph_new cimport *
+import cugraph.structure.graph_new_wrapper as graph_new_wrapper
+from libc.stdint cimport uintptr_t
+from cython.operator cimport dereference as deref
+
+def mg_pagerank(input_df, local_data, handle, alpha=0.85, max_iter=100, tol=1.0e-5):
```

Comment:
https://github.com/rapidsai/cugraph/blob/aeee8aae1f72fdb823c235ae57f927dd4be4cb47/python/cugraph/snmg/link_analysis/mg_pagerank.py#L15

Should we get rid of this? It could be confusing since they are called the same.

Reply:
https://github.com/rapidsai/cugraph/pull/944#discussion_r441028764, It is to let the user know that snmg is disabled. We could remove it once OPG is in place, or, move the exception throwing at python/cugraph/dask/pagerank/pagerank.py